contribs/gmf/src/directives/mobilenav.js

```diff
@@ -243,6 +243,56 @@ gmf.MobileNavController.prototype.backIfActive = function(element) {
 
 /**
  * A directive to be used in conjunction with {@link gmf.mobileNavDirective}.
+ * The directive can be set on a slide element of {@link gmf.mobileNavDirective}
+ * with an expression. When the value of the expression changes and becomes
+ * true, the navigation returns to the previous slide, if the slide is
+ * currently active.
+ *
+ * Example:
+ *
+ *    <nav class="nav-left" gmf-mobile-nav>
+ *      ...
+ *      <gmf-themeselector
+ *         id="themes"
+ *         class="slide"
+ *         data-header-title="Themes"
+ *         gmf-themeselector-currenttheme="mainCtrl.theme"
+ *         gmf-mobile-nav-back="mainCtrl.theme">
```

Comment:
Should be changed to something like `gmf-mobile-nav-back="mainCtrl.theme.name !== 'OSM'"` (boolean value)